src/06-sort.js

```diff
@@ -23,17 +28,35 @@ function sortByRuntimeAscending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortByArtistNameDescending(songs) {}
+function sortByArtistNameDescending(songs) {
+  return songs.sort((a, b) => {
+    if(a.artist.toLowerCase() > b.artist.toLowerCase())
+     return -1
+     if(a.artist.toLowerCase() < b.artist.toLowerCase())
+     return 1
+     if(a.artist.toLowerCase() === b.artist.toLowerCase())
+     return 0
+  })
+}
```

Comment:
You should really always use curly brackets to demarcate when an `if` block begins and ends, but ESPECIALLY when you're putting the 1 line of code that you're allowed to do that with underneath. It makes it looks like the next line after it is still part of the if's block! And the indentation leads to that confusion as well.

Better to have the curly braces and make it IMMEDIATELY obvious what will execute if your condition is true and what won't. Better overly clear than less clear but shorter.